packages/reactstrap-validation-select/AvSelect.js

```diff
@@ -171,6 +172,53 @@ class AvSelect extends AvBaseInput {
       }
     }
     super.onChangeHandler(value);
+
+    const shouldAutofill =
+      this.props.autofill &&
+      !this.props.isMulti &&
+      inputValue &&
+      typeof inputValue === 'object';
+
+    if (shouldAutofill) {
+      const formValues = this.context.FormCtrl.getValues();
+      Object.keys(formValues)
+        // Filter out the input that the onChangeHandler is being called for
+        .filter(fieldName => fieldName !== name)
+        .forEach(fieldName => {
+          let rawValue = inputValue;
+          if (
+            !!inputValue.label &&
+            !!inputValue.value &&
+            typeof inputValue.value === 'object'
+          ) {
+            rawValue = inputValue.value;
+          }
+
+          let inputValueIsInForm = false;
+          if (typeof this.props.autofill === 'object') {
+            inputValueIsInForm = Object.keys(this.props.autofill).some(
```

Comment:
Not 100% sure what this is doing. Wouldn't something like `this.props.autofill[fieldName]` determine the same thing?
If would filter things that wouldn't be too useful in the `get` below. And if you want to keep those things, `Object.prototype.hasOwnProperty.call(this.props.autofill, fieldName)` would also work (or just lodash's `has`).

Reply:
The purpose of that is to check if that form input has a corresponding key in `autofill` (if `autofill` is an object) or in the `rawValue` (if `autofill` is a bool). If it does, we update the form input with the autofilled value. Maybe `inputValueIsInForm` would be better named `autofillKeyIsInForm`. 

`this.props.autofill[fieldName]` would also work - but not for the `else` statement. we would still want to autofill the component if `rawValue[fieldName]` is falsy. I'll update.

I don't understand what you mean in this part:

> If would filter things that wouldn't be too useful in the get below. And if you want to keep those things, Object.prototype.hasOwnProperty.call(this.props.autofill, fieldName) would also work (or just lodash's has).

Reply:
Please see e34cac45ed3ee41d79b6effafb50767b4e7abd97 . Thanks for feedback!

Reply:
For that other part, I mean if `this.props.autofill[fieldName]` is `false`, the `.some` would still return `true` since the key exists, but the value `false` in the `get` wouldn't really work...

Reply:
Makes sense now. Added 4740d97b56dd63865ea391f8cda992e714993118 Thanks again! 🙌